### ISSUE TITLE
sysinternals@February.18.2019: hash check failed

### ISSUE THREAD

**Issue by sys32592** [lukesampson/scoop-extras#1884]
I tried to install sysinternals,

>$ scoop install sysinternals
>Installing 'sysinternals' (February.18.2019) [64bit]
>Downloading https://download.sysinternals.com/files/SysinternalsSuite.zip (24.2 MB)...
>Checking hash of SysinternalsSuite.zip ... ERROR Hash check failed!
>App:         extras/sysinternals
>URL:         https://download.sysinternals.com/files/SysinternalsSuite.zip
>First bytes: 50 4B 03 04 14 00 00 00
>Expected:    9b9d6bd0718a5680b2322398345271142783156dc8d6f3642bce72f80337850f
>Actual:      aa9bf58a84c78503c1e5feae9f0dce8f6867ebdf9cea1956564d051f4bcd651c

>Please try again or create a new issue by using the following link and paste your console output:
>https://github.com/lukesampson/scoop-extras/issues/new?title=sysinternals%40February.18.2019%3a+hash+check+failed

Checking out the Microsoft web site 
https://docs.microsoft.com/en-us/sysinternals/downloads/sysinternals-suite
It appears the download site only keeps the latest version of the zip file available.
When I manually download the file, and unzipped it, I can see the zip file contains files dated after Feb 18, 2019.

>$ ls -lt
>total 88154
>-rw-r--r-- 1 eric 197121 25427414 Mar 19 20:22 SysinternalsSuite.zip
>-rwxr-xr-x 1 eric 197121  3012656 Mar 16 14:41 Bginfo.exe*
>-rwxr-xr-x 1 eric 197121  4228352 Mar 16 14:40 Bginfo64.exe*
>-rwxr-xr-x 1 eric 197121  3309304 Mar 11 12:59 Sysmon.exe*
>-rwxr-xr-x 1 eric 197121  1813552 Mar 11 12:55 Sysmon64.exe*
>-rwxr-xr-x 1 eric 197121   444984 Mar  4 10:54 psloglist.exe*
>-rwxr-xr-x 1 eric 197121   579128 Mar  4 10:52 psloglist64.exe*

A quick search for versions of sysinternalssuite shows that chocolatey thinks sysinternalssuite was updated on March 18, 2019, which is consistent with the dates I see in the zip file.
>https://chocolatey.org/packages/sysinternals

A sha256sum of the SysInternalSuite.zip file I downloaded from 
matches the value that scoop reports when it downloaded.

>$ sha256sum SysinternalsSuite.zip
>aa9bf58a84c78503c1e5feae9f0dce8f6867ebdf9cea1956564d051f4bcd651c *SysinternalsSuite.zip

Bottom line, I think the SysInternalsSuite has been updated and the scoop description is referencing an old version that is no longer available at the download site. 

Please let me know if I can do anything to help out. I really appreciate all the effort that goes into scoop. It makes my life better. :-)

**Comment by r15ch13** [comment]
Fixed with https://github.com/lukesampson/scoop-extras/pull/1892
